unifi_discovery/__init__.py

```
@@ -199,37 +310,51 @@ def parse_ubnt_response(
+    for field_type, field_data in iter_fields(payload[4:], data_len):
+        if field_type not in field_parsers:
             continue
 
         # Parse the field and store in Device
-        field_name, field_parser, is_many = field_parsers_packet_specific[field_type]
-        value = field_parser(field_data)  # type: ignore
+        field_name, field_parser, is_many = field_parsers[field_type]
+        try:
+            value = field_parser(field_data)  # type: ignore
+        except Exception:
+            continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)